build.ps1

```diff
@@ -27,6 +33,52 @@ param(
 
 #Requires -Version 6.0
 
+$PowerShellVersion = '7'
+$TargetFramework = 'netcoreapp3.1'
+
+function Get-FunctionsCoreToolsDir {
+    if ($CoreToolsDir) {
+        $CoreToolsDir
+    } else {
+        $funcPath = (Get-Command func).Source
+        if (-not $funcPath) {
+            throw 'Cannot find "func" command. Please install Azure Functions Core Tools.'
```

Comment:
Please add the link to the Core Tools installation instructions, e.g., https://github.com/Azure/azure-functions-core-tools#installing

Reply:
Added, thank you